packages/web/stores/ibc-notifier.tsx

```diff
@@ -21,6 +21,8 @@ export const IbcNotifier: FunctionComponent = observer(() => {
     if (!historyHandlerAdded) {
       ibcTransferHistoryStore.addHistoryChangedHandler((history) => {
         if (chainStore.hasChain(history.destChainId)) {
+          console.log("history", history);
```

Comment:
Note: This line is being triggered twice per IBC transfer. I'm failing to detect what is the retigger